docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,375 @@
+</ReferenceTable>
+
+### Working directory
+
+Use the `working_directory` setting to load Dagster code from a different directory than the root of your code repository. This setting allows you to specify the directory you want to load your code from.
+
+Consider the following project:
+
+```shell
+quickstart_etl
+├── README.md
+├── project_directory
+│   ├── quickstart_etl
+│     ├── __init__.py
+│     ├── assets
+│   ├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+To load from `/project_directory`, the `dagster_cloud.yaml` code location would look like this:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="working_directory">
+    The path of the directory that Dagster should load the code source from
+    <ul>
+      <li>
+        <strong>Format</strong> - <code>string</code>(path)
+      </li>
+    </ul>
+  </ReferenceTableItem>
+</ReferenceTable>
+
+
+### Build
+
+The build registry is applicable to hybrid deployments only. Setting a build directory is useful if your `setup.py` or `requirements.txt` is in a subdirectory instead of the project root. This is common if you have multiple Python packages within a single Dagster project.
+
+In the example below, the Docker image for the code location is in the root directory and the registry and image defined:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    build:
+      directory: ./
+      registry: your-docker-image-registry/image-name # e.g. localhost:5000/myimage
+```
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="build.directory">
+    The path to the directory in your project that you want to deploy. If there
+    are subdirectories, you can specify the path to only deploy a specific
+    project directory.
+    <ul>
+      <li>
+        <strong>Format</strong> - <code>string</code>(path)
+      </li>
+      <li>
+        <strong>Default</strong> - <code>.</code>
+      </li>
+    </ul>
+  </ReferenceTableItem>
+  <ReferenceTableItem propertyName="build.registry">
+    <strong>Applicable to Hybrid deployments.</strong> The Docker registry to
+    push your code location to in the registry
+    <ul>
+      <li>
+        <strong>Format</strong> - <code>string</code>(docker registry)
+      </li>
+    </ul>
+  </ReferenceTableItem>
+</ReferenceTable>
+
+### Python executable
+
+For Dagster Cloud Hybrid deployments, the Python executable that is installed globally in the image, or the default Python executable on the local system if you use the local agent, will be used. To use a different Python executable, specify it using the python_executable setting. It can be useful to have different Python executables for different code locations.
```

Comment:
```suggestion
For Dagster Cloud Hybrid deployments, the Python executable that is installed globally in the image, or the default Python executable on the local system if you use the local agent, will be used. To use a different Python executable, specify it using the `python_executable` setting. It can be useful to have different Python executables for different code locations.
```